ozaria/site/components/cinematic/common/CinematicCanvas.vue

```diff
@@ -1,9 +1,15 @@
 <template>
   <!-- TODO: Canvas needs to be responsive to scaling up and down. -->
   <!-- Currently fixed size to the aspect ratio of our play view. -->
-  <div>
-    <div height="514px" id="cinematic-div" ref="cinematic-div" v-on:click="skipShot">
-      <canvas width="800" height="514" id="cinematic-canvas" ref="cinematic-canvas"></canvas>
+  <div id="cinematic-canvas-div">
+    <div height="850px" id="cinematic-div" ref="cinematic-div" v-on:click="skipShot">
+      <canvas
+        id="cinematic-canvas"
+        ref="cinematic-canvas"
+        :width="width"
+        :height="height"
+        :style="{ width: width+'px', height: height+'px' }">
```

Comment:
Is `style` required here with `width` and `height` being there individually as well?

Reply:
It is required because the canvas width and canvas height it the number of pixels on the canvas.
The style width and style height is how large we display it.

For example. If we wanted retina quality canvas, we could set the width and height at twice the width and height. But then keep the css the same. This would compact more pixels into the space giving higher quality.
I wasn't able to make this work as the maths in the camera doesn't work out and I'm scared of breaking it. It also appears a high cost.

Reply:
Sounds good, thanks for explaining!